src/components/WeatherWidget/WeatherWidget.jsx

```diff
@@ -0,0 +1,132 @@
+import { StyledWrapper, StyledWrapperWeather } from "./WeatherWidget.styles";
+import { useState, useEffect } from "react";
+import GetLocation from "../../services/location";
+import GetWeather from "../../api/weather";
```

Comment:
Variables starting with capital letters are reserved for components only, so these would be:

```suggestion
import { getLocation } from "../../services/location";
import { getWeather } from "../../api/weather";
```

I've also made them named imports as opposed to default imports (so please change in corresponding files accordingly), as this would be more natural. If you were to add any new method, you won't have to change the default to named, etc.